lib/clients/git-client.js

```diff
@@ -36,12 +36,13 @@ module.exports = class GitClient {
     /**
      * Config local git credential helper with credentialHelperPath
      * @param {string} credentialHelperPath the path of git credential helper
+     * @param {string} hostUrl host url to apply credential helper to
      */
-    configureCredentialHelper(credentialHelperPath) {
+    configureCredentialHelper(credentialHelperPath, hostUrl) {
         const commands = [
-            'git config --local credential.helper ""',
-            `git config --local --add credential.helper "!${credentialHelperPath}"`,
-            'git config --local credential.UseHttpPath true'];
+            `git config --local credential.${hostUrl}.helper ""`,
```

Comment:
not sure if we need this line

Reply:
Yes, I think so, it's used to clean up the helper list 
"If credential.helper is configured to the empty string, this resets the helper list to empty " from https://www.git-scm.com/docs/gitcredentials#Documentation/gitcredentials.txt-helper